pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+testpaths = /Users/kindra/dev/rosetta-io
```

Comment:
Is this needed? I didn't need to do this on my machine. But, if it is, then maybe we should figure out a way to not hardcode it (because I don't have the same username as you)

Reply:
No, you're right, it's not needed. Deleting